docs/index.rst

```diff
@@ -24,6 +24,38 @@ database using the `PostgreSQL Wire Protocol`_.
 databases in Java.
 
 
+*****
+About
+*****
+
+Overview
+========
+
+- This JDBC driver is needed in certain scenarios like the one outlined at
+  `Apache Kafka, Apache Flink, and CrateDB`_.
+- Officially, and for general purpose use, we recommend to use the canonical
+  `PostgreSQL JDBC Driver`_ instead.
+
+.. _implementations:
+.. _jdbc-implementation:
+
+What's inside
+=============
+
+The driver is based upon a fork of the `PostgreSQL JDBC Driver`_, see
+`pgjdbc driver fork`_.
+
+Please take notice of the corresponding implementation notes:
+
+- `CallableStatement`_ is not supported, as CrateDB itself does not support
+  stored procedures.
+- `DataSource`_ is not supported.
+- `ParameterMetaData`_, e.g. as returned by `PreparedStatement`_, is not
+  supported.
+- `ResultSet`_ objects are read only (``TYPE_FORWARD_ONLY``, ``CONCUR_READ_ONLY``),
+  so changes to a ``ResultSet`` are not supported.
```

Comment:
I have not verified this, but I think it will be true?

Reply:
yep, you cannot write to the result set.